pkg/ttnpb/errors.go

```diff
@@ -50,8 +51,26 @@ func (e errorDetails) Cause() error {
 	return errorDetails{cause}
 }
 
+func (e errorDetails) Details() []proto.Message {
+	details := e.GetDetails()
+	if len(details) == 0 {
+		return nil
+	}
+
+	msgs := make([]proto.Message, 0, len(details))
+	for _, dAny := range details {
+		var msg types.DynamicAny
+		err := types.UnmarshalAny(dAny, &msg)
+		if err != nil {
+			panic(fmt.Sprintf("Failed to decode error details: %s", err))
```

Comment:
I very much dislike the idea of `panic`ing here, but implemented it this way to be consistent with https://github.com/TheThingsNetwork/lorawan-stack/pull/995/files#diff-dc97abe08c40b3ea0c5eb2da325331a8R41

Reply:
I agree. Please open a technical debt issue for that.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/issues/1023